lsp-dart.el

```diff
@@ -80,6 +80,154 @@ Defaults to 0.9"
   :group 'lsp-dart
   :package-version '(lsp-mode . "6.3"))
 
+(defcustom lsp-dart-outline t
+  "When set to non-nil, dart/textDocument/publishOutline notifications will
+be sent with outline information for open files. Defaults to t"
+  :type 'boolean
+  :group 'lsp-dart
+  :package-version '(lsp-mode . "6.3"))
+
+(defcustom lsp-dart-flutter-outline t
+  "When set to non-nil, dart/textDocument/publishFlutterOutline notifications will
+be sent with Flutter outline information for open files. Defaults to t"
+  :type 'boolean
+  :group 'lsp-dart
+  :package-version '(lsp-mode . "6.3"))
+
+(defcustom lsp-dart-outline-position-params
+  `((side . ,treemacs-position)
+    (slot . 2)
+    (window-width . ,treemacs-width))
+  "The outline tree position params.
+Defaults to side following treemacs default."
+  :type 'list
+  :group 'lsp-dart
+  :package-version '(lsp-mode . "6.3"))
+
+(defcustom lsp-dart-flutter-outline-position-params
+  `((side . ,treemacs-position)
```

Comment:
you cannot use treemacs-position here like that - since we do not have a dependency on it.

Reply:
Wrapped into a `with-eval-after-load`